libcontainer/cgroups/systemd/v2.go

```diff
@@ -292,6 +296,32 @@ func (m *unifiedManager) Apply(pid int) error {
 	if err := fs2.CreateCgroupPath(m.path, m.cgroups); err != nil {
 		return err
 	}
+
+	goodToChown := []string{"cgroup.procs", "cgroup.subtree_control", "cgroup.threads"}
+	member := func(haystack []string, needle string) bool {
+		for _, v := range haystack {
+			if v == needle {
+				return true
+			}
+		}
+		return false
+	}
+	if m.uid >= 0 {
+		err := filepath.Walk(m.path, func(path string, info os.FileInfo, err error) error {
```

Comment:
Since you're only using info.Name(), it probably make sense to switch to filepath.WalkDir. Problem is, it is only added in go 1.16. So maybe add a TODO item:
```go
// TODO: switch to filepath.WalkDir once go < 1.16 is not supported.
```

Reply:
Yep, I originally used WalkDir then had to change for that reason.  Happy to add the comment.